src/algebra/star/self_adjoint.lean

```diff
@@ -54,6 +57,17 @@ def skew_adjoint [add_comm_group R] [star_add_monoid R] : add_subgroup R :=
 
 variables {R}
 
+/-- An element of a star monoid is normal if it commutes with its adjoint. -/
+class is_star_normal [has_mul R] [has_star R] (x : R) : Prop :=
+(star_comm_self_aux : commute (star x) x)
```

Comment:
Why have the `aux`?
```suggestion
(star_comm_self : commute (star x) x)
```

Reply:
I was having trouble with rewriting at some point when I just had `export` but I don't remember exactly what the problem was and I can't reproduce it now.

Reply:
I just changed it back.